clarity_ext/dilution.py

```diff
@@ -79,27 +78,38 @@ def _build_plate_sorting_map(containers):
         return plate_position_numbers
 
     def __str__(self):
-        return "<{type} {robot} {rows}x{cols}>".format(type=self.__class__.__name__,
-                                                       robot=self.robot_name,
-                                                       rows=self.plate_size_y,
-                                                       cols=self.plate_size_x)
-
-PLATE_TYPE_96_WELL = 1
```

Comment:
That sounds good. To me it doesn't matter if it's row/column or height/width as long as it's clear what it means - but x/y became rather confusing. Using what the lab is most used to is preferable.

Since it's not entirely related to this pull request (I'm just using the plate object rather than the property), I've added a ticket on this.